app/views/ctc/ctc_pages/home.html.erb

```diff
@@ -70,9 +69,15 @@
       </div>
       <div class="grid__item width-one-half">
         <h2><%= t('views.ctc_pages.home.what_is.heading') %></h2>
-        <% t('views.ctc_pages.home.what_is.body').each do |body| %>
-          <p><%= body %></p>
-        <% end %>
+        <%= t('views.ctc_pages.home.what_is.body') %>
+        <div class="reveal-shrink-wrapper spacing-above-15">
+          <%= render('components/molecules/reveal', title: t("views.ctc_pages.home.what_is.qualify_reveal.title")) do %>
```

Comment:
<img width="857" alt="Screen Shot 2022-04-29 at 4 04 12 PM" src="https://user-images.githubusercontent.com/49880002/166061805-f9bd239e-721f-478f-9494-91cdad242e3c.png">
adds reveals and new phones image